src/main/resources/db/changelog/changes/03-remove-delete-column-from-book-table.yaml

```diff
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
+      id: remove-delete-column-from-book-table
+      author: gulini
+      changes:
+        - dropColumn:
```

Comment:
dropColumn isn't supported auto rollback
So rename field to `isDeleted` in 01-create-books-table.yaml

Or you can also implement rollback approach